src/connection.js

```diff
@@ -81,18 +82,10 @@ module.exports = class Connection extends EventEmitter {
       callback(
         null,
         peersAddresses
-          .filter(
-            (peerAddress) => peerAddress.peer.id.toB58String() === peerId)
-          .map(peerAddress => peerAddress.addr)
+        .filter(
+          (peerAddress) => peerAddress.peer.id.toB58String() === peerId)
+        .map(peerAddress => peerAddress.addr)
```

Comment:
since you're filtering and mapping `peerAddresses`, I think this should remain indented to denote that.